unit_test/CMakeLists.txt

```diff
@@ -14,8 +17,8 @@ KOKKOSKERNELS_INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_C
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/batched)
 INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR}/batched)
 
-KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/../test_common)
-KOKKOSKERNELS_INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR}/../test_common)
+KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/graph)
```

Comment:
Should this line include ``${CMAKE_CURRENT_BINARY_DIR}/common`` instead? .../graph is included up on line 14

Reply:
I removed that, this was not making sense as pointed out.